SyslogNet.Client/Transport/SyslogLocalSender.cs

```diff
@@ -117,6 +118,16 @@ public void Send(IEnumerable<SyslogMessage> messages)
 			Send(messages, defaultSerializer);
 		}
 
+		public Task SendAsync(SyslogMessage message, ISyslogMessageSerializer serializer)
+		{
+			throw new NotImplementedException();

```

Comment:
Returning a `TaskCompleted` to provide some implementation for this method will make the users unexpectedly use blocking code in their programs. 

Check an infamous example of the frustration of developers upon the unexpected discovery of this same problem in one of MS libraries here: https://connect.microsoft.com/VisualStudio/feedback/details/779079/console-readlineasync-is-actually-blocking-when-it-should-be-returning-a-task-immediately